tfx/proto/infra_validator.proto

```diff
@@ -119,6 +119,46 @@ message KubernetesConfig {
   //
   // Should be a positive integer. Default to 24 hours (86400).
   int32 active_deadline_seconds = 2;
+
+  // Optional.
+  // Additional Kubernetes pod overrides to apply to the serving pod
+  PodManifestOverrides serving_pod_manifest_overrides = 3;
+}
+
+message PodManifestOverrides {
+  // Optional.
+  // Kubernetes annotations to apply to the pod
```

Comment:
nit: For comments we have a rule that every sentence (even single) should end with period, and the number of columns doesn't exceed 80. Would you apply such lint for the CL?

Reply:
Fixed this.